packages/access-api/src/routes/validate-email.js

```diff
@@ -134,6 +139,93 @@ async function session(req, env) {
     req.query.ucan,
     delegation.capabilities[0].nb.key
   )
+  if (req.method.toLowerCase() === 'post') {
+    const accessSessionResult = await validator.access(delegation, {
+      capability: Access.session,
+      principal: Verifier,
+      authority: env.signer,
+    })
+    if (accessSessionResult.error) {
+      throw new Error(
+        `unable to validate access session: ${accessSessionResult.error}`
+      )
+    }
+    const accountDID = accessSessionResult.audience.did()
+    // @todo: use new ucanto `Account` instead
+    const accountOneOffKey = await ed25519.generate()
+    const account = accountOneOffKey.withDID(accountDID)
+
+    const agentPubkey = accessSessionResult.capability.nb.key
+
+    const update = await ucanto.delegate({
```

Comment:
It would really help to add comment above explaining what this delegation intends to do. In the version that I think this implements this delegation is attestation from us that account has authorized agent key to be able to sign ucans where issuer is account did.